src/ansible_navigator/_version.py

```diff
@@ -1,7 +1,6 @@
-""" version
+"""Version information for ansible-navigator.
 
 Note:
-
```

Comment:
I wouldn't remove this line. Other vars below have a separate empty line before them. Plus if you use an admonition, the whole block would need to be indented.

Reply:
the others have a blank after, not before :)

Reply:
put back